packages/server/src/api/events/events.service.ts

```diff
@@ -1258,15 +1258,41 @@ export class EventsService {
         _id: event.correlationValue,
         workspaceId,
       }, {
-        other_ids: { $in: [event.correlationValue] },
+        other_ids: event.correlationValue,

```

Comment:
No, we don't. We can just run a query with `other_ids: UUID` and it will try to find `UUID` in the array elements. Check out the docs for more info: https://www.mongodb.com/docs/manual/tutorial/query-arrays/#query-an-array-for-an-element

The way we were doing it, we are trying to find `other_ids` in an array containing one value: `event.correlationValue`